File: spring-cloud/config-server/src/test/java/fr/edjaz/springcloud/configserver/ConfigServerApplicationTests.java
package fr.edjaz.springcloud.configserver;

import org.junit.jupiter.api.Test;
import org.junit.jupiter.api.extension.ExtendWith;
import org.springframework.boot.test.context.SpringBootTest;
import org.springframework.test.context.junit.jupiter.SpringExtension;

import static org.springframework.boot.test.context.SpringBootTest.WebEnvironment.RANDOM_PORT;

@ExtendWith(SpringExtension.class)
@SpringBootTest(webEnvironment=RANDOM_PORT, properties = {"spring.profiles.active=native"})
class ConfigServerApplicationTests {

	@Test
	void contextLoads() {
	}

}

File: spring-cloud/config-server/src/test/kotlin/fr/edjaz/springcloud/configserver/ConfigServerApplicationTests.kt
package fr.edjaz.springcloud.configserver

import org.junit.jupiter.api.Test
import org.junit.jupiter.api.extension.ExtendWith
import org.springframework.boot.test.context.SpringBootTest
import org.springframework.boot.test.context.SpringBootTest.WebEnvironment
import org.springframework.test.context.junit.jupiter.SpringExtension

@ExtendWith(SpringExtension::class)
@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, properties = ["spring.profiles.active=native"])
class ConfigServerApplicationTests {
    @Test
    fun contextLoads() {
    }
}
